### PR TITLE
Update web_sql_injection_in_access_logs.yml

### PR DESCRIPTION
**Pull request by ThureinOo**
### Summary of the Pull Request
This RP has updated detection method for some bypass techinques.

### Changelog
update: Added some bypass methods used by SQLI Injectors. 

### Example Log Event
N/A

### Fixed Issues
N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by nasbench** [comment]
@ThureinOo :) please add everything you want to add in a single PR instead of creating many. It will ease the review process.

Also add references/examples that reflect your additions.

As I commented on your previous PR, don't add any strings, please choose carefully and describe in your PR message the reason behind some of the addition

**Comment by ThureinOo** [comment]
I added some extra functions that I've used to bypass the filters when I study SQLI Techniques. That's why i cannot provide public references for them. Thank you so much for your understanding. I am making some mistakes as I have no prior experience in this.

**Comment by nasbench** [comment]
No worries. I'll look more into it :) and see how can we move this forward.

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 1]
```diff
@@ -46,6 +46,11 @@ detection:
         - 'select version()'
         - 'select%20database()'
         - 'select%20version()'
+        - 'json_arrayagg('
+        - 'from mysql.innodb_table_status'
+        - 'from%20mysql.innodb_table_status'
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:51]:
```suggestion
        - 'from mysql.innodb_table_stats'
        - 'from%20mysql.innodb_table_stats'
```

File: rules/web/webserver_generic/web_sql_injection_in_access_logs.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ references:
     - https://www.acunetix.com/blog/articles/using-logs-to-investigate-a-web-application-attack/
     - https://brightsec.com/blog/sql-injection-payloads/
     - https://github.com/payloadbox/sql-injection-payload-list
```
Comment (nasbench) [inline rules/web/webserver_generic/web_sql_injection_in_access_logs.yml:9]:
```suggestion
    - https://github.com/payloadbox/sql-injection-payload-list
    - https://book.hacktricks.xyz/pentesting-web/sql-injection/mysql-injection
```

**Review by nasbench — APPROVED** [review]

**Review by phantinuss — APPROVED** [review]

**Pull request merged**